Lib/fontMath/test/test_mathGlyph.py

```diff
@@ -509,6 +509,12 @@ def test_image_round(self):
         glyph2 = glyph1.round()
         self.assertEqual(glyph2.image, expected)
 
+    def test_copy(self):
+        glyph1 = self._setupTestGlyph()
+        glyph1.unicodes = []
```

Comment:
acutally, not so sure about that, because I see elsewhere checks like `if self.unicodes is not None` etc. so maybe that attribute is supposed to be optional.. no idea

Reply:
I think the real bug is that `self.unicodes` is initialized to `None` in `MathGlyph.__init__`. Out of scope for this PR perhaps.

Reply:
This test is basically for codecov to shut up and make the CI pass... It barely tests anything useful.